deployment/ansible/roles/model-my-watershed.geoprocessing/templates/upstart-akka-http.conf.j2

```diff
@@ -0,0 +1,13 @@
+description "Akka HTTP Server"
+
+start on runlevel [3]
```

Comment:
Seems like this could also just be `runlevel [2]`: http://upstart.ubuntu.com/cookbook/#runlevels